css/blocks/team.css

```diff
@@ -0,0 +1,68 @@
+.team {
+  padding: 94px 0 94px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  background: rgba(245, 244, 250, 1);
+}
+
+.team__title {
+  margin: 0 0 50px;
+  text-align: center;
+  font-size: 36px;
+  font-weight: 700;
```

Comment:
in all section titles, styles are similar. It would make more sense not to repeat same rules for each section title, but rather create one element section__title and pass all the styles to it, and use this class on all section titles.

* one recommendation - change font-weight to 500 as 700 thorughout the projects, as 700 is too bold and doesn't look nice